plugins/hls-notes-plugin/test/NotesTest.hs

```diff
@@ -16,19 +16,20 @@ main = defaultTestRunner $
 
 gotoNoteTests :: TestTree
 gotoNoteTests = testGroup "Goto Note Definition"
-    [ testCase "single_file" $ runSessionWithServer def plugin testDataDir $ do
+    [
+      testCase "single_file" $ runSessionWithServer def plugin testDataDir $ do
         doc <- openDoc "NoteDef.hs" "haskell"
         waitForKickDone
         defs <- getDefinitions doc (Position 3 41)
         liftIO $ do
-          fp <- canonicalizePath "NoteDef.hs"
+          fp <- canonicalizePath $ testDataDir </> "NoteDef.hs"
```

Comment:
We should probably add a `Todo` here, is the `canonicalizePath` even necessary?

Reply:
Let remove it and pass the the root instead.